test/testlib/minikube_utilities.go

```diff
@@ -836,6 +836,12 @@ func GetAdminEventLog(t *testing.T, namespace string, podName string) {
 	)
 }
 
+func AwaitPodLog(t *testing.T, namespaceName string, podName string, fileNameSuffix string) {
```

Comment:
this function does not block until the log is collected. It's a goroutine. Is that what you intended?

Reply:
The main purpose of the function is to wait until pod is running and then to request its log. The actual log collection is non-blocking. If you request the log before that, it will report that either the pod or container is not running and will skip the log collection.